src/components/DownloadsComponents/DownloadsList.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import DownloadItem from './DownloadItem';
+
+export default function DownloadsList({ items }) {
+	// placeholder for multiple downloads
+	return items.map((item) => <DownloadItem />);
```

Comment:
I think you forgot the [`key` prop](https://reactjs.org/docs/lists-and-keys.html#keys).